Atividades/para-casa-roberta/exercicio.js

```diff
@@ -0,0 +1,165 @@
+class AnimalDomestico {
+  constructor(nome, idade, cor, castrado) {
+    this.nome = nome;
+    this.idade = idade;
+    this.cor = cor;
+    this.consultas = [];
+    this.vacinas = [];
+    this.castrado = castrado;
+  }
+}
+
+class Gato extends AnimalDomestico {
+  constructor(nome, idade, cor, castrado, externo, social) {
+    super(nome, idade, cor, castrado);
+    this.externo = externo;
+    this.social = social;
+    this.consultas = [];
+    this.vacinas = [];
+  }
```

Comment:
Eu acho que não precisaria repetir o "this.consultas" e o "this.vacinas", porque essas informações já estão dentro de "AnimalDomestico".

Não citei novamente no meu e deu certo, mas se tá certo ou não, aí não sei dizer kkkk